span-normalizer/span-normalizer-constants/src/main/java/org/hypertrace/core/semantic/convention/constants/error/OTelErrorSemanticConventions.java

```diff
@@ -4,7 +4,11 @@
 public enum OTelErrorSemanticConventions {
   EXCEPTION_TYPE("exception.type"),
   EXCEPTION_MESSAGE("exception.message"),
-  EXCEPTION_STACKTRACE("exception.stacktrace");
+  EXCEPTION_STACKTRACE("exception.stacktrace"),
+  STATUS_CODE("status.code"),
+  STATUS_CODE_UNSET_VALUE("0"),
```

Comment:
This might be confusing since 0 usually means success and 1 error for status codes, no?

Reply:
Yah, but, this is how it is defined in proto - https://github.com/open-telemetry/opentelemetry-proto/blob/abbf7b7b49a5342d0d6c0e86e91d713bbedb6580/opentelemetry/proto/trace/v1/trace.proto#L304

Reply:
ok.